src/dbmate/build.py

```diff
@@ -0,0 +1,25 @@
+from ..buildlib import build_wheels
+
+NAME = 'dbmate'
+VERSION = '2.17.0'
+PYPI_VERSION = '2.17.0'
```

Comment:
Can we make the initial PYPI_VERSION `2.17.0a1` so that we can do an initial alpha release in case it needs to be yanked, since this is the first attempt at publishing this :-)